src/Umbraco.Web.UI.Client/src/packages/documents/documents/workspace/views/info/document-workspace-view-info-links.element.ts

```diff
@@ -75,24 +79,38 @@ export class UmbDocumentWorkspaceViewInfoLinksElement extends UmbLitElement {
 				this.#setLinks();
 			});
 		});
+
+		this.consumeContext(UMB_APP_LANGUAGE_CONTEXT, (instance) => {
+			this.observe(instance.appDefaultLanguage, (value) => {
+				this._defaultCulture = value?.unique;
+				this.#setLinks();
+			});
+		});
 	}
 
 	#setLinks() {
-		const possibleVariantCultures = this._variantOptions?.map((variantOption) => variantOption.culture) ?? [];
-		const possibleUrlCultures = this.#urls.map((link) => link.culture);
-		const possibleCultures = [...new Set([...possibleVariantCultures, ...possibleUrlCultures])].filter(
-			Boolean,
-		) as string[];
-
-		const links: Array<UmbDocumentInfoViewLink> = possibleCultures.map((culture) => {
-			const url = this.#urls.find((link) => link.culture === culture)?.url;
+		const links: Array<UmbDocumentInfoViewLink> = this.#urls.map((u) => {
+			const culture = u.culture ?? this._defaultCulture ?? "";
+			const url = u.url;
 			const state = this._variantOptions?.find((variantOption) => variantOption.culture === culture)?.variant?.state;
 			return { culture, url, state };
 		});
 
 		this._links = links;
 	}
 
+	#getTargetUrl (url: string | undefined) {
+		if (!url || url.length === 0) {
+			return url;
+		}
+
+		if (url.includes(".") && !url.includes("//")) {
+			return "https://" + url;
```

Comment:
I guess it would be safer to assume non-SSL protocol here as web hosts most likely would have added some kind of automatic redirect if they support HTTPS. If we force https and the host does not support it, then the user gets a blank page.

Reply:
Or perhaps just `//`?  Then that would take whatever the backoffice is running on.

Reply:
Have done that - but can amend if you think otherwise.

Reply:
Even better with `//` 👍